src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -3,6 +3,8 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
     release_level: BETA
+    class_names:
+      google.example.library.v1.LibraryService: LibraryClient
```

Comment:
The settings class isn't getting renamed. The two of them need to be in sync.

I suggest a different approach where the base name is given. Something like:

```
  interface_names:
    google.example.library.v1.LibraryService: Library
```

And this generates `LibraryClient` and `LibrarySettings` for classes, and `libraryClient` and `librarySettings` for variable names.